xml/cl.xml

```diff
@@ -7412,30 +7449,31 @@ server's OpenCL/api-docs repository.
             <require>
                 <type name="cl_platform_command_buffer_capabilities_khr"/>
             </require>
-            <require comment="cl_platform_info">
+            <require group="cl_platform_info">
                 <enum name="CL_PLATFORM_COMMAND_BUFFER_CAPABILITIES_KHR"/>
             </require>
-            <require comment="cl_platform_command_buffer_capabilities_khr - bitfield">
+            <require group="cl_platform_command_buffer_capabilities_khr" etype="bitfield">
                 <enum name="CL_COMMAND_BUFFER_PLATFORM_UNIVERSAL_SYNC_KHR"/>
                 <enum name="CL_COMMAND_BUFFER_PLATFORM_REMAP_QUEUES_KHR"/>
                 <enum name="CL_COMMAND_BUFFER_PLATFORM_AUTOMATIC_REMAP_KHR"/>
             </require>
-            <require comment="cl_device_info">
+            <require group="cl_device_info">
+                <enum name="CL_DEVICE_COMMAND_BUFFER_CAPABILITIES_KHR"/>
```

Comment:
I don't think this is correct - `CL_DEVICE_COMMAND_BUFFER_CAPABILITIES_KHR` was added by `cl_khr_command_buffer` and is not part of `cl_khr_command_buffer_multi_device`.

```suggestion
            <require group="cl_device_info">
```

Reply:
I can't seem to find why I added it here... Well, let me just commit this and see what might break.